components/url_sanitizer/resources/default_trackers.json

```diff
@@ -0,0 +1,64 @@
+[
```

Comment:
I guess we should be able to delete this file now?

Reply:
done